cmd/server-main.go

```diff
@@ -129,6 +131,13 @@ var ServerFlags = []cli.Flag{
 		Name:  "sftp",
 		Usage: "enable and configure an SFTP server",
 	},
+	cli.IntFlag{
```

Comment:
For an experiment it is fine, but a `--numa="node=1;mem=1,2"` would be more flexible.

Reply:
Correct but didn't want to bring in memory bind as well here. Since by default membind indeed is is spanning across.

Reply:
@harshavardhana It may make sense in some setups. We have only really tested drive read speeds. For most data operations (hashing/erasure coding/general copying) will be much slower if memory is on different cores - though L3 may be able to make up for it.

Just saying that a more flexible param may be more future proof, if we want to add it officially.

(you can resolve for now)

Reply:
Yeah I will bring membind as well